src/scaffolding/graph.ts

```diff
@@ -71,8 +72,8 @@ export async function getAddGraphKeyPayload(
   ];
 
   const graphKeyAction: any = {
-    targetHash: 0,
-    schemaId: 7,
+    targetHash,
+    schemaId: publicGraphKeySchema.id,
```

Comment:
Does this not having a default cause any issues if no value is passed? Id there ever a case where no value is passed?

Reply:
I mean... the only gotcha is that you must call `fetchPublicKeySchema` at some point first before calling this method. Might be worth mentioning in comments. We could use put in a fall-back default value, but I think it's better to have this fail. We're supposed to resolve the schema on-chain first, and if we don't all bets should be off.